unity-client/Assets/Scripts/MainScripts/DCL/Components/Video/Tests/VideoTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿using DCL;
+using DCL.Helpers;
+using DCL.Components;
+using DCL.Models;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using DCL.Controllers;
+
+namespace Tests
+{
+    public class VideoTests : TestsBase
+    {
+
+        protected override IEnumerator SetUp()
+        {
+            yield return base.SetUp();
+            DCLVideoTexture.isTest = true;
+        }
+
+        [UnityTest]
+        public IEnumerator VideoTextureIsAttachedAndDetachedCorrectly()
+        {
+            // TEST: DCLVideoTexture is created correctly
```

Comment:
Instead of using comments, those test segments should be promoted to a full test each